library/platform_util.c

```diff
@@ -129,6 +129,15 @@ void mbedtls_platform_zeroize(void *buf, size_t len)
 }
 #endif /* MBEDTLS_PLATFORM_ZEROIZE_ALT */
 
+void mbedtls_zeroize_and_free(void *buf, size_t len)
```

Comment:
Would it be desirable to define as `mbedtls_zeroize_and_free(void **buf, size_t len)` so that this function can set the pointer to NULL?

Reply:
Changing the pointer is helpful if there's a single copy of the pointer, but misleading if there are multiple copies. It's not something we do at the moment, and I think if we do this it's best to do it consistently so that we're used to it when reviewing. So I prefer not to do it.

Reply:
That seems reasonable, just wanted to raise the question.